cypress/integration/ete-okta/registration_2.6.cy.ts

```diff
@@ -832,6 +832,48 @@ describe('Registration flow - Split 2/2', () => {
 			);
 		});
 
+		it('passcode used functionality', () => {
+			const unregisteredEmail = randomMailosaurEmail();
+			cy.visit(`/register/email?usePasscodeRegistration=true`);
+
+			const timeRequestWasMade = new Date();
+			cy.get('input[name=email]').type(unregisteredEmail);
+			cy.get('[data-cy="main-form-submit-button"]').click();
+
+			cy.contains('Enter your code');
+			cy.contains(unregisteredEmail);
+			cy.contains('send again');
+			cy.contains('try another address');
+
+			cy.checkForEmailAndGetDetails(unregisteredEmail, timeRequestWasMade).then(
+				({ body, codes }) => {
+					// email
+					expect(body).to.have.string('Your verification code');
+					expect(codes?.length).to.eq(1);
+					const code = codes?.[0].value;
+					expect(code).to.match(/^\d{6}$/);
+
+					// passcode page
+					cy.get('input[name=code]').clear().type(code!);
+					cy.contains('Submit verification code').click();
+
+					cy.url().should('contain', '/welcome/password');
+
+					cy.go('back');
+					cy.url().should('contain', '/register/email');
```

Comment:
Why not check that `passcodeUsed` is in the URL here? The other tests check that the page looks right, but it'd be also good to have that checked.

Reply:
because it's not in the url, it's in the encrypted state cookie which we extract into the page data